packages/@sanity/desk-tool/src/utils/withInitialValue.js

```diff
@@ -160,4 +158,18 @@ function resolveInitialValueWithParameters(templateName, parameters) {
   )
 }
 
+function resolvePaneOptions(props, templateName) {
+  const {options} = props
+  const hasDefinedType = options.type && options.type !== '*'
+  const typeFromOptions = hasDefinedType ? options.type : undefined
+  let documentType = typeFromOptions || (document && document._type)
```

Comment:
I must be blind or something, but I can't see how `document` is available in this function?

Reply:
Well spotted!